.golangci.yml

```diff
@@ -28,7 +28,7 @@ linters:
 issues:
   exclude-rules:
     # Excluding single digits from magic number detector because it produces too many obvious results (like klog)
-    - text: "Magic number: [0-9]{1},"
+    - text: "Magic number: [0-9]{1,2},"
```

Comment:
Can you please revert this and just add

```
//nolint:gomd // <comment here>
```

to lines 151 and 152 in lballoc.go?

In general, we want 2 digit numbers to be flagged because they often hide things that should be variables. In your case, they're obvious, but the lint is helpful to have for other users.

Reply:
Done